docs/changelog/87430.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 87430
```

Comment:
We can delete this changelog entry. This isn't a user-facing chnage so we typically don't include build changes in our release notes.

Reply:
Sounds good! I added it because of the checks below asking for it but I just erased it!